src/components/Compare/Compare.style.tsx

```diff
@@ -30,6 +30,13 @@ const getMetricHeaderBackground = (
   }
 };
 
+export const StyledModal = styled(Modal)`
```

Comment:
nit: "StyledModal" is a bit generic, especially since there are 2 different modal usages in here.  So maybe "FaqModal" or "CenteredContentModal" or something?